_redirects

```diff
@@ -1,4 +1,5 @@
 # See:  https://docs.netlify.com/routing/redirects/
 #       https://docs.netlify.com/routing/redirects/redirect-options/
 /schemas/latest/* /schemas/0.5/:splat    200
-/schemas/snapshot/* /schemas/0.6-alpha/:splat    200
\ No newline at end of file
+/schemas/snapshot/* /schemas/0.6-alpha/:splat    200
+/services/diagrams/generate https://diagram-service-secure-mmagnani.apps.kogito-cloud.automation.rhmw.io/swdiagram 200
```

Comment:
could this be like "/services/diagram"
only? i doubt we will have anything else except "generate".

sorry :)

Reply:
Sure, no problem!